libraries/goclient/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/intel/afxdp-plugins-for-kubernetes/libraries/goclient
+
+go 1.18
```

Comment:
I think we can move this down to 1.13 - I'm not 100% sure but I believe it enforces the minimum version of Go that can import and use your library, so I think we can lower it to make it more usable.